content/participants/ttjc@Uttkarsh-Srivastava/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Utkarsh Srivastava
+type: "participant"
+name: "dummy student"
```

Comment:
please change the name from dummy student to your name

Reply:
@gkanishk changed the name. Thank you for pointing out the error.